usecases/ast_eval/evaluate/evaluate_aggregator.go

```diff
@@ -100,18 +100,11 @@ func (a AggregatorEvaluator) runQueryInRepository(ctx context.Context, tableName
 		return DryRunQueryAggregatedValue(a.DataModel, tableName, fieldName, aggregator)
 	}
 
-	return transaction.InOrganizationSchema_deprec(
-		ctx,
-		a.OrgTransactionFactory,
-		a.OrganizationId,
-		func(tx repositories.Transaction_deprec) (any, error) {
-			result, err := a.IngestedDataReadRepository.QueryAggregatedValue(ctx, tx, tableName, fieldName, aggregator, filters)
-			if err != nil {
-				return nil, err
-			}
-			return result, nil
-		},
-	)
+	if db, err := a.ClientSchemaExecutorFactory.NewClientDbExecutor(ctx, a.OrganizationId); err != nil {
+		return nil, err
+	} else {
+		return a.IngestedDataReadRepository.QueryAggregatedValue(ctx, db, tableName, fieldName, aggregator, filters)
+	}
```

Comment:
useless else (syntax nazy here, but I like the simple `if / return` structure of go which limit nesting

Reply:
hmm yea that was in order to keep the "if err!=nil" condition more inline, but in the end I agree I gain nothing from it